Packs/QRadar/ReleaseNotes/2_2_0.md

```diff
@@ -0,0 +1,13 @@
+
+#### Integrations
+##### IBM QRadar v3
+- Improved mirroring performance.
+- Added the number of fetched events in the offense context.
+
+#### Mappers
+##### QRadar - Generic Incoming Mapper
+- Fixed the mapping of the number of fetched events.
```

Comment:
Please add a comment about it previously showing lower numbers than actually fetched.